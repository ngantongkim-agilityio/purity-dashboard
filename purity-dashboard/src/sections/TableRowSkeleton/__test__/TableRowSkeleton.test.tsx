import { render } from '@testing-library/react';

// Components
import { TableRowSkeleton } from '..';

describe('TableRowSkeleton component', () => {
  it('Should render TableRowSkeleton component with default props', () => {
    const { container } = render(<TableRowSkeleton />);

    expect(container).toMatchSnapshot();
  });
});
