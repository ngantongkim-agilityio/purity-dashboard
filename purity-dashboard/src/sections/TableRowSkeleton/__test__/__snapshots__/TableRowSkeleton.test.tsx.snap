// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableRowSkeleton component Should render TableRowSkeleton component with default props 1`] = `
<div>
  <tr
    class="w-full border-b border-gray-100 last-of-type:border-none [&:first-child>td:first-child]:rounded-tl-lg [&:first-child>td:last-child]:rounded-tr-lg [&:last-child>td:first-child]:rounded-bl-lg [&:last-child>td:last-child]:rounded-br-lg"
  >
    <td
      class="relative overflow-hidden whitespace-nowrap py-3 pl-6 pr-3"
    >
      <div
        class="flex items-center gap-3"
      >
        <div
          class="h-8 w-8 rounded-full bg-gray-100"
        />
        <div
          class="h-6 w-24 rounded bg-gray-100"
        />
      </div>
    </td>
    <td
      class="whitespace-nowrap px-3 py-3"
    >
      <div
        class="h-6 w-32 rounded bg-gray-100"
      />
    </td>
    <td
      class="whitespace-nowrap px-3 py-3"
    >
      <div
        class="h-6 w-16 rounded bg-gray-100"
      />
    </td>
    <td
      class="whitespace-nowrap px-3 py-3"
    >
      <div
        class="h-6 w-16 rounded bg-gray-100"
      />
    </td>
    <td
      class="whitespace-nowrap px-3 py-3"
    >
      <div
        class="h-6 w-16 rounded bg-gray-100"
      />
    </td>
    <td
      class="whitespace-nowrap py-3 pl-6 pr-3"
    >
      <div
        class="flex justify-end gap-3"
      >
        <div
          class="h-[38px] w-[38px] rounded bg-gray-100"
        />
        <div
          class="h-[38px] w-[38px] rounded bg-gray-100"
        />
      </div>
    </td>
  </tr>
</div>
`;
